klaviyo/public.py

```diff
@@ -63,8 +65,7 @@ def track(
         if ip_address:
             params['ip'] = ip_address
 
-        query_string = self._build_query_string(params, is_test)
-        return self._public_request(self.TRACK, query_string)
+        return self._public_post_request(method=method, params=params, path=self.TRACK, is_test=is_test)
```

Comment:
this method name doesn't make sense here, it could be either a get or post but it looks like the method always says post

Reply:
updated